idaes/core/util/tables.py

```diff
@@ -266,22 +267,47 @@ def stream_table_dataframe_to_string(stream_table, **kwargs):
         na_rep=na_rep, justify=justify, float_format=float_format, **kwargs
     )
 
+def _get_state_from_port(port,time_point):
+    """
+    Attempt to get a state-block-like object connected to a Port. If different
+    variables on the port appear to be connected to different state blocks,
+    raise an exception. This process is complicated by the fact that the object
+    may be indexed by space as well as time. Therefore, we assume that, if it
+    is, the time index comes first.
+    """
```

Comment:
Another minor point, but it might be good to document the arguments, returns and raises characteristics of this method. Whilst it is currently private, I could see this being promoted to a public method in the not too distant future.

Either way, it would be useful for future developers to know what errors are raised by different conditions so that they can add error handling to their methods as necessary.